cmd/snappy/common.go

```diff
@@ -52,12 +52,12 @@ func withMutexAndRetry(f func() error) error {
 		// if already locked, auto-retry
 		if err == priv.ErrAlreadyLocked {
 			var msg string
-			if isAutoPilotRunning() {
+			if isAutoUpdateRunning() {
 				// FIXME: we could even do a
 				//    journalctl -u snappy-autopilot
 				// here
 				msg = i18n.G(
-					`The snappy autopilot is updating your system in the background. This may
+					`snappy autoupdate is updating your system in the background. This may
```

Comment:
Same.. why do we need to call out "snappy autoupdate" as an entity? Why doesn't that say "snappy is updating your system in the background."?